concourse/tasks/delete-ssh-keys.yml

```diff
@@ -0,0 +1,21 @@
+---
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: governmentpaas/awscli
+inputs:
+  - name: paas-cf
+run:
+  path: sh
+  args:
+    - -e
+    - -c
```

Comment:
re: -u
https://github.com/alphagov/paas-cf/pull/802/files#diff-8f7658bb02a08dfb771f38694a7b075fR2272
I actually need it. I can change it do full "prefix" - that is, the whole name of key.